people_options.rb

```diff
@@ -0,0 +1,58 @@
+require './person'
+require './student'
+require './teacher'
+
+class PeopleOptions
+  attr_reader :people
+
+  def initialize
+    @people = []
+  end
+
+  # List all people.
+  def list_all_people
+    @people.each_with_index do |person, index|
+      puts "#{index + 1})  [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
```

Comment:
- [ ] (Optional) It would be nice to print a message that says no person is available if a `student` or a `teacher` has not been created. 👍